`core.zig`  and `template.zig` should be included in the release tarball

I modified `example/application` `build.zig` file to use the released bindings as follows:
```
.{
    .name = "application",
    .version = "1.0.0",
    .paths = .{
        "build.zig",
        "build.zig.zon",
        "application.zig",
        "example_app_prefs.zig",
        "example_app_window.zig",
        "example_app.zig",
    },
    .dependencies = .{
        .gtk = .{
          .url = "https://github.com/DerryAlex/zig-gir-ffi/releases/download/v0.8.1/gtk4.tar.gz",
          .hash = "1220025b8d2c4b6d199a83c7ad4d63fc1b4f303c54b939861a29d30d0dc2e692dd9f",
        },
    },
}
```
and I got the following error when tried to do `zig build`:
```
install
└─ install application
   └─ zig build-exe application Debug native 2 errors
/home/hasan/.cache/zig/p/1220025b8d2c4b6d199a83c7ad4d63fc1b4f303c54b939861a29d30d0dc2e692dd9f/Gtk.zig:17:26: error: unable to load '/home/hasan/.cache/zig/p/1220025b8d2c4b6d199a83c7ad4d63fc1b4f303c54b939861a29d30d0dc2e692dd9f/core.zig': FileNotFound
pub const core = @import("core.zig");
                         ^~~~~~~~~~
/home/hasan/.cache/zig/p/1220025b8d2c4b6d199a83c7ad4d63fc1b4f303c54b939861a29d30d0dc2e692dd9f/Gtk.zig:16:30: error: unable to load '/home/hasan/.cache/zig/p/1220025b8d2c4b6d199a83c7ad4d63fc1b4f303c54b939861a29d30d0dc2e692dd9f/template.zig': FileNotFound
pub const template = @import("template.zig");
                             ^~~~~~~~~~~~~~
```
Which I was able to solve by copying manual/* files to the cache folder

--- a/build.zig
+++ b/build.zig
@@ -7,7 +7,7 @@ pub fn build(b: *std.Build) !void {
     const xml = b.dependency("xml", .{});
 
     const options = b.addOptions();
-    const version = b.option([]const u8, "version", "application version string") orelse "0.8.1";
+    const version = b.option([]const u8, "version", "application version string") orelse "0.8.2";
     options.addOption([]const u8, "version", version);
 
     const exe = b.addExecutable(.{

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -1,6 +1,6 @@
 .{
     .name = "zig-gir-ffi",
-    .version = "0.8.1",
+    .version = "0.8.2",
     .paths = .{
         "build.zig",
         "build.zig.zon",

--- a/src/main.zig
+++ b/src/main.zig
@@ -51,6 +51,8 @@ pub fn main() !void {
         \\    .paths = .{{
         \\        "build.zig",
         \\        "build.zig.zon",
+        \\        "core.zig",
+        \\        "template.zig",
         \\
     , .{version});
 

